frontend/sozisel-app/src/components/utils/Typescript/ensure.ts

```diff
@@ -0,0 +1,10 @@
+export function ensure<T>(
```

Comment:
Do you really need that kind of a function? Looks like something taken out of reanimated where you try to convince typescript about js values.

Reply:
Yes, and I think it might be useful.

Reply:
Ok, if you say so.